internal/dcache/cluster_manager/cluster_manager.go

```diff
@@ -1849,12 +1849,12 @@ func (cmi *ClusterManager) updateMVList(rvMap map[string]dcache.RawVolume,
 			}
 
 			//
-			// If this component RV is not offline, its containing node must be excluded for replacement RV(s).
+			// If this component RV is not offline/inband-offline, its containing node must be excluded for replacement RV(s).
 			// We don't exclude the node if the component RV is offline to support the case where the same node
 			// comes back up online and we may want to use the same RV or another RV from the same node, as
 			// replacement RV.
 			//
-			if mv.RVs[rvName] != dcache.StateOffline {
+			if mv.RVs[rvName] != dcache.StateOffline && mv.RVs[rvName] != dcache.StateInbandOffline {
```

Comment:
actually we should leave it as before since for inband-offline we would like to exclude the node as most likely other RVs on that node are also not reachable. If we allow that node to be used for picking replacement RVs, we might get into a situation where it picks another RV from the same node and that's bad too, then next time around it picks this one and so on..

Reply:
Done